spec.html

```diff
@@ -33431,7 +33431,7 @@ <h1>String.prototype.lastIndexOf ( _searchString_ [ , _position_ ] )</h1>
           1. If _searchStr_ is the empty String, return 𝔽(_start_).
           1. Let _searchLen_ be the length of _searchStr_.
           1. For each non-negative integer _i_ starting with _start_ such that _i_ &le; _len_ - _searchLen_, in descending order, do
-            1. Let _candidate_ be the substring of _string_ from _i_ to _i_ + _searchLen_.
+            1. Let _candidate_ be the substring of _searchStr_ from _i_ to _i_ + _searchLen_.
```

Comment:
```suggestion
            1. Let _candidate_ be the substring of _s_ from _i_ to _i_ + _searchLen_.
```
Surely? You're searching for _searchStr_ in _S_.

Reply:
(－‸ლ) yes, thank you. this is why review is helpful